modules/c++/sys/include/sys/AbstractOS.h

```diff
@@ -231,6 +231,12 @@ class AbstractOS
      */
     virtual void getMemInfo(size_t& totalPhysMem, size_t& freePhysMem) const = 0;
 
+    /*!
+     *  Get the absolute path to the current executable
```

Comment:
For all new functions added, use Doxygen comments.  Namely, put in each `\param` and a `\return`